crates/ruff_graph/src/lib.rs

```diff
@@ -91,17 +91,24 @@ impl ModuleImports {
 
 #[derive(Debug, Default)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct ImportMap(BTreeMap<SystemPathBuf, ModuleImports>);
+pub struct ImportMap(BTreeMap<SystemPathBuf, BTreeSet<SystemPathBuf>>);
```

Comment:
Can we name the fields? The code is now kind of hard to understand where the types are generic data structures and the fields are not named

Reply:
You could also use File as values if you happen to have interned anyway. 

A File is just a u32, so super fast to hash

Reply:
What kinds of names would help here? Do you mean using named type aliases or named fields?

Reply:
Oh wow, smart. I do need it to be sorted / ordered in the output though 🤔

Reply:
I could try doing everything on u32 and then converting to sorted system paths at the very end

Reply:
(I just retained the use of the struct here.)